libs/styles/src/lib/scss/components/_metadata-card.scss

```diff
@@ -0,0 +1,33 @@
+@mixin cvi-metadata-card($base: &) {
+  --comment: 'A card with a coloured left border and content';
+
+  display: flex;
+  border-radius: var(--cvi-radius-s);
+
+  @include cvi-breakpoint-up(sm) {
+    box-shadow: var(--cvi-shadow-beta);
+  }
+
+  #{$base}__left-border {
+    width: 6px;
+    background-color: var(--left-border-color);
```

Comment:
It's better (and conventional to the coding style of the project - see eg `cvi-button`) to prefix a CSS variable meant for customisation in a component with the name of that component. 

This is because a CSS variable like this could be set to a global selector such as `:root` or `body`, and we don't want it to be overridden with a same-name variable for another component.